st3/sublime_lib/_util/collections.py

```diff
@@ -35,30 +46,30 @@ def projection(d, keys):
         }
 
 
-def get_selector(selector, default_value=None):
+def get_selector(selector: object, default_value: object = None) -> Callable:  # noqa: F811
```

Comment:
`object` is the top type, whereas `Any` disables type checking on that value. `object` is appropriate here because we can't do anything to the args until we've narrowed the type down manually in the function.

Most occurrences of `Any` in this PR should probably be removed, though it's probably not a blocker.

Reply:
Thanks, this comment prompted me to do some research and [the explanation in PEP 484](https://www.python.org/dev/peps/pep-0484/#the-any-type) made things quite clear. While both `Any` and `object` apply to all types, one is the higher and the other is the lower end of abstraction. 

Since we only perform operations onto what a basic object can do and don't have any further restrictions, we want `object` here.